src/base/static/components/ui-elements/tertiary-button.js

```diff
@@ -0,0 +1,16 @@
+import React, { Component } from "react";
+
+const baseClass = "tertiary-button";
+
+class TertiaryButton extends Component {
```

Comment:
I suggest making this a functional component, ie:

```
const TeritiaryButton = () => (
  <button ....
      ...
  </button>
)
```

It's easier to read, and better communicates that the component doesn't carry any state. And for React <16, they were more performant, but I don't think that's the case anymore for React 16.